README-gke.md

```diff
@@ -16,11 +16,7 @@ Status](https://travis-ci.org/lsst/qserv_deploy.svg?branch=master)](https://trav
    export QSERV_CFG_DIR="$HOME/.qserv/"
    mkdir -p $QSERV_CFG_DIR
 
-   # Please ask support for how to fine tune files below
-   # (This files should be deprecated in near future)
-   cp -r "qserv_deploy/config.examples/petasky* "$QSERV_CFG_DIR"
-
-   cp <kubeconfig> $QSERV_CFG_DIR
+   cp -r "qserv_deploy/config.examples/gke "$QSERV_CFG_DIR"
```

Comment:
Found a bug while running this, there should be no quote before qserv_deploy:

`cp -r "qserv_deploy/config.examples/gke "$QSERV_CFG_DIR`